paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PaymentMethodOrderSettingsDefinition.kt

```diff
@@ -0,0 +1,28 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.example.playground.PlaygroundState
+
+internal object PaymentMethodOrderSettingsDefinition :
+    PlaygroundSettingDefinition<String>,
+    PlaygroundSettingDefinition.Saveable<String>,
+    PlaygroundSettingDefinition.Displayable<String> {
+    override val key: String = "paymentMethodOrder"
+    override val displayName: String = "Payment method order"
+    override val defaultValue: String = ""
```

Comment:
Would it be better to represent the default value as null here?

Reply:
This is our example playground representation of how to build a `PaymentSheet.Configuration`. The default value here represents what we put in a text field as the default value for our playground. Representing this as a string probably isn't ideal, but it's the best way we can do "free form" input for customizing behavior of things like this.